src/slash_command.ts

```diff
@@ -65,24 +66,36 @@ export async function handleSlashCommand(
         for (let key in variables) {
             if (key !== 'env_vars') {
                 env_vars[key] = variables[key];
-                allSet &&= await tfcApi.setVariable(workspaceId, existingVars[key], key, variables[key]);
```

Comment:
I removed workspaceId as an argument to `setVariable` since workspace IDs are unique to TFC. Instead `TerraformCloudAPI#setVariable(...)` manages the step of retrieving the TFC workspace ID.